client/src/components/Navbar/Navbar.tsx

```diff
@@ -126,7 +125,11 @@ const Navbar: React.FC = () => {
     handleClose();
     logout();
   };
-  const [profile, setProfile] = useState<Profile>({ accountType: 'initial' });
+
+  interface IAccountType {
```

Comment:
```suggestion
  interface AccountType {
```
Please avoid using 'I' as a prefix for interfaces in JS